### ISSUE TITLE
Address various a11y violations in pages

### ISSUE THREAD

**Issue by nicolethoen** [patternfly/patternfly-org#3123]
- [ ] Utilities > Sizing - images must have alt text
- [ ] ignore axe rules `landmark-main-is-top-level`, `landmark-no-duplicate-main`
- [ ] update navToggleId in pageHeader in sideNavLayout

issues to fix on training pages:
- [ ]  iframes need accessible labels
 - [ ] Scrollable regions must have keyboard access

issues to fix in design guidelines:
 - [ ] images need alt text or accessible labels
 - [ ] Tree view design guidelines
 - [ ] Primary detail demo design guidelines
 - [ ] Inline edit design guidelines
 - [ ] Modal design guidelines - aria-label cannot be used on an i element

**Comment by nicolethoen** [comment]
Closing in favor of the renewed effort captured in this issue: https://github.com/patternfly/patternfly-org/issues/3618
